bedrock/mozorg/templates/mozorg/diversity/2021/index.html

```diff
@@ -0,0 +1,510 @@
+{#
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "base-protocol-mozilla.html" %}
+
+{% from "macros.html" import image with context %}
+{% from "macros-protocol.html" import split, picto, card, feature_card, with context %}
+
+{% block page_title %}D&I report 2021{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('diversity-2021') }}
+{% endblock %}
+
+{% block js %}
+  {{ js_bundle('diversity-2021') }}
+{% endblock %}
+
+{% block sub_navigation %}
+<nav class="c-sub-navigation">
+  <div class="mzp-l-content">
+    <div class="c-sub-navigation-content">
+      <h2 class="c-sub-navigation-title">Diversity & Inclusion 2021</h2>
+      <ul class="c-sub-navigation-list">
+        <li class="c-sub-navigation-item"><a href="{{ url('mozorg.diversity.2021.index') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Diversity and inclusion overview">Overview</a></li>
+        <li class="c-sub-navigation-item"><a href="{{ url('mozorg.diversity.2021.who-we-are') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Who we are">Who we are</a></li>
+        <li class="c-sub-navigation-item"><a href="{{ url('mozorg.diversity.2021.what-we-build') }}" data-link-type="nav" data-link-position="subnav" data-link-name="What we build">What we build</a></li>
+        <li class="c-sub-navigation-item"><a href="{{ url('mozorg.diversity.2021.beyond-products') }}" data-link-type="nav" data-link-position="subnav" data-link-name="Beyond our products">Beyond our products</a></li>
+      </ul>
+    </div>
+  </div>
+</nav>
+{% endblock %}
+
+{% block content %}
+
+<main role="main">
+  <header class="c-hero-banner">
+    <div class="mzp-l-content">
+      <div class="c-banner-content">
+        <div class="c-header-container">
+          <h1>
+            Diversity & Inclusion at Mozilla: 2021 Results
+          </h1>
+        </div>
+      </div>
+    </div>
+  </header>
```

Comment:
**nit(non-blocking)**: Looks like there is some extra spacing in between the header and the main body - looks like mzp-l-content might be the reason why